redi-connect-front/src/components/atoms/FormDatePicker.tsx

```diff
@@ -22,21 +20,42 @@ const PickerTrigger = ({ value, onClick }: any) => (
     <Icon
       icon="calendar"
       className="datepicker-trigger__icon"
-      size="medium" />
+      size="medium"
+    />
   </div>
 )
 
-function FormDatePicker (props: any) {
+interface IFormDatePicker {
```

Comment:
Please refactor name to `FormDatePickerProps`. We want to follow the convention established by other "props interfaces". To quickly locate other examples, run this search on the `redi-connect-front` folder, with regex turned on:
`\}:.+\)`

Reply:
Other than that: really nice stuff!